tests/testthat/test-run-general.R

```diff
@@ -1256,7 +1256,7 @@ test_that("user parameter validation", {
 })
 
 test_that("force integer on use", {
-  gen <- odin({
+  gen <- odin_js({
```

Comment:
oh, were we testing the C version before?

Reply:
yeah, in porting the tests over the function was never updated :see_no_evil: